src/go/k8s/tests/e2e/schema-registry-tls-client-external-ca/00-assert.yaml

```diff
@@ -0,0 +1,101 @@
+apiVersion: redpanda.vectorized.io/v1alpha1
+kind: Cluster
+metadata:
+  name: schema-registry-client-external-ca-mtls
+status:
+  replicas: 1
+
+---
+
+apiVersion: v1
+kind: Secret
+metadata:
+  name: schema-registry-client-external-ca-mtls-schema-registry-trusted-client-ca
+
+---
+
+apiVersion: v1
+kind: Secret
+metadata:
+  name: schema-registry-client-cert-key
+
+---
+
+apiVersion: v1
+kind: Secret
+metadata:
+  name: schema-registry-client-self-signed-cert-key
+
+---
+
+apiVersion: cert-manager.io/v1
+kind: Issuer
+metadata:
+  name: schema-registry-client-external-ca-mtls-schema-registry-selfsigned-issuer
+status:
+  conditions:
+    - reason: IsReady
+      status: "True"
+      type: Ready
+
+---
+
+apiVersion: cert-manager.io/v1
+kind: Issuer
+metadata:
+  name: schema-registry-client-external-ca-mtls-schema-registry-root-issuer
+status:
+  conditions:
+    - reason: KeyPairVerified
+      status: "True"
+      type: Ready
+
+---
+
+apiVersion: cert-manager.io/v1
+kind: Certificate
+metadata:
+  #name: schema-registry-client-external-ca-mtls-schema-registry-root-certificate
+  name: schema-registry-client-external-schema-registry-root-certificate
+status:
+  conditions:
+    - reason: Ready
+      status: "True"
+      type: Ready
+
+---
+
+apiVersion: cert-manager.io/v1
+kind: Certificate
+metadata:
+  name: schema-registry-client-external-ca-mtls-schema-registry-node
+status:
+  conditions:
+    - reason: Ready
+      status: "True"
+      type: Ready
+
+---
+
+apiVersion: redpanda.vectorized.io/v1alpha1
+kind: Cluster
+metadata:
+  name: schema-registry-client-external-ca-mtls
+status:
+  readyReplicas: 1
+  replicas: 1
+  upgrading: false
+  restarting: false
+  conditions:
+  - type: ClusterConfigured
+    status: "True"
```

Comment:
Can this move to the top of this file and combine with the spec https://github.com/redpanda-data/redpanda/pull/9863/files#diff-5e25ad94d2a14d457eaf68b504506c9fdb29a0a363eeeab8eb75b4d35d751335R1-R6?

Reply:
Done.